src/en/space-station-14/player-interaction/proposals/powered-hardsuits.md

```diff
@@ -0,0 +1,97 @@
+
+## Several possible scenarios
+
+I would like to demonstrate the idea behind this proposal and the effects it can have on the game by giving four scenarios where hardsuit abilities interact with the rest of the game as a whole while adhering to the core game design pillars.
+
+- Imagine: A syndicate agent is using an engineering hardsuit for the welding protection ability, they have the plan to deconstruct into the armory to steal from it, so they also wear a pair of sunglasses. They have now attained the abilities of both welding goggles and sunglasses, but due to them leaving the welding goggle ability on after they were done using it, they run out of hardsuit power while fleeing security in space and are either apprehended or purchase a blood-red hardsuit/syndicate EVA suit from their uplink on the move. 
+
+- Imagine: An engineer is repairing a major spacing near cargo, but his suit runs out of power. He asks salvage if he can use one of their suit storages to charge, which they allow him to do so by opening the door. While the suit is charging, the salvager walks out of the room, leaving the engineer locked inside cargo to either call for the AI, hope the salvager returns, or repeatedly fail to open the door hoping the QM or a different salvager notices.
+
+- Imagine: Engineering is having trouble managing their suit charge constantly throughout the station during a survival/kessler round, so science constructs various suit storages around the station near commonly damaged areas. Engineering uses them often but is left vulnerable to having their suit stolen by a tider or the clown due to the public area.
+
+- Imagine: Nukies are taking a very long time to complete their mission, and their suits begin to run low on power. To conserve charge, they start using their suit abilities less. Possibly including noslips that they didn't otherwise buy allowing the clown to slip them, and turning off lights allowing the crew to sneak up on them. Or they decide to take a break and charge their suits at the crew's suit storages, leaving them vulnerable to a surprise attack, or even possibly retreating to their ship to charge using storages provided there, giving the crew a chance to recover and regroup for a counter-attack.
+
+## Required changes
+- Suit storages would now be a machine rather than just a plain closet. Likely with a single slot for a suit so you can't charge 7 at once. 1-2 oxygen tank slots or gas mask slots would be good as well. You should still be able to hide in them though like you can in a closet.
+- EVA storage uses closets/lockers instead of suit storages to prevent immediate easy access to hardsuit charging in the middle of the station.
```

Comment:
Meeeeeeeeeeeh, maybe? You kinda have to ask the HoP for this, maybe map a few. I'm indifferent to it being included/not included but otherwise just something to consider.

You can always map in some more if there's a problem with people complaining in the first 2 weeks (although this will probably happen anyways until people adjust to a different play style compared to old suits)

Reply:
> Actually, no you don't lol. Most maps allow sec/engi to get into eva without hop/command letting them in.

Ah, yeah you're right, on some maps there's an engineering pass through. When I play CE it feels like I never have inherent access to EVA though, but there are some odd ones out. Only map I can think of that has EVA set to externals access is Cog.

> I do think having a couple suit storages here or there could be fine on some maps, but ideally nearly all suit storages should be either roundstart in a department that uses hardsuits, or built by science to improve efficiency.

Agree. If science wants more work done, they build more (and maybe even better better?) chargers.